wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -127,7 +137,7 @@ public function update( $new_instance, $instance ) {
 	 * @return void
 	 */
 	public function render_media( $instance ) {
-		$instance = array_merge( $this->default_instance, $instance );
+		$instance = array_merge( wp_list_pluck( $this->get_instance_schema(), 'default' ), $instance );
```

Comment:
@obenland yes, your right. But since there's only a couple calls I thought maybe it would be better keep it DRY instead of duplicate the data in two places.